src/main/java/com/nike/cerberus/domain/environment/Environment.java

```diff
@@ -123,23 +75,10 @@ public Environment setConfigKeyId(String configKeyId) {
         return this;
     }
 
-    public boolean isCd() {
-        return isCd;
-    }
-
-    public Environment setCd(boolean cd) {
-        isCd = cd;
-        return this;
-    }
-
     public Map<String, BackupRegionInfo> getRegionBackupBucketMap() {
         return regionBackupBucketMap == null ? new HashMap<>() : regionBackupBucketMap;
     }
 
-    public void setRegionBackupBucketMap(Map<String, BackupRegionInfo> regionBackupBucketMap) {
```

Comment:
I know this probably wasn't used, but do we want to use it?

Reply:
No, we get the map that already exists and just add items to it and then save